x-pack/plugin/security/build.gradle

```diff
@@ -257,7 +267,13 @@ thirdPartyAudit {
         'net.sf.ehcache.Ehcache',
         'net.sf.ehcache.Element',
         // [missing classes] SLF4j includes an optional class that depends on an extension class (!)
-        'org.slf4j.ext.EventData'
+        'org.slf4j.ext.EventData',
+        'org.cryptomator.siv.SivMode',
+        'org.objectweb.asm.ClassWriter',
+        'org.objectweb.asm.Label',
+        'org.objectweb.asm.MethodVisitor',
+        'org.objectweb.asm.Type'
+
```

Comment:
Can we put a comment in here about what these classes are and why we're ignoring the violations?

Reply:
Thanks for keeping me honest here, turns out that `org.ow2.asm:asm:7.1` is a legitimate transient compile dependency and shouldn't be excluded